tests/ui/lint/use-redundant/use-redundant-prelude-rust-2015.rs

```diff
@@ -2,11 +2,15 @@
 #![warn(unused_imports)]
 
 
-use std::option::Option::Some;//~ WARNING the item `Some` is imported redundantly
-use std::option::Option::None; //~ WARNING the item `None` is imported redundantly
+use std::option::Option::Some;
+//FIXME(unused_imports): ~^ WARNING the item `Some` is imported redundantly
```

Comment:
Line break b/c line was too long; alternatively could throw out these altogether, but it's nice to keep em around imo.